server/channels/api4/file_test.go

```diff
@@ -832,9 +838,9 @@ func TestGetFileHeaders(t *testing.T) {
 	t.Run("txt", testHeaders(data, "test.txt", "text/plain", false, false))
 	t.Run("html", testHeaders(data, "test.html", "text/plain", false, false))
 	t.Run("js", testHeaders(data, "test.js", "text/plain", false, false))
-	if os.Getenv("IS_CI") == "true" {
+	if os.Getenv("IS_CI") == "true" || runtime.GOOS == "darwin" {
```

Comment:
If this is inconsistent why are we testing its behavior? Should we just remove the test like below for the `exe` test? Save us from more headache when the behavior changes again.